src/models/User.js

```diff
@@ -9,7 +9,11 @@ const UserSchema = new Schema(
     nickname: { type: String, required: false },
     email: { type: String, required: true, unique: true },
     password: { type: String, required: true },
-    avatar: { type: String, required: true, default: 'yxz' },
+    avatar: {
+      type: String,
+      required: false,
+      default: './statics/avatar/01.png',
+    },
```

Comment:
are we supposed to see this image by default in the front-end? I am just getting text.

Reply:
we will after i insert the statics path